src/app/Http/Controllers/UserController.php

```diff
@@ -299,6 +300,22 @@ public function saveCourse(Request $request): JsonResponse
         }
     }
 
+    /**
+     * @return JsonResponse
+    */
+    public function getUserCourses() : JsonResponse
+    {
+        $user = Auth::user();
+
+        $user_courses = $user->courses()->get();
+
+        if(!$user->courses()->exists()){
+            $user_courses = Course::available()->get(['id', 'type', 'name', 'can_be_applied_until', 'start_at']);
+        }
```

Comment:
```suggestion
        $user_courses = Auth::user()->courses()->get();
```

Kullanıcının tüm kurslarını döneceğimiz için direk böyle çekip dönebiliriz.
get() zaten aktif kursu yoksa boş array verecek bize, bu yüzden exists kontrolünede gerek yok.
tek sorguyla halletmiş oluruz.